src/lib/votesReport.ts

```diff
@@ -43,9 +43,9 @@ class VotesReport extends Cache {
 
     const headers = [
       'address',
-      votes.length === 0 || typeof votes[0].choice === 'number'
+      ['basic', 'single-choice'].includes(this.proposal!.type)
         ? 'choice'
-        : this.proposal && this.proposal.choices.map((_choice, index) => `choice.${index + 1}`),
+        : this.proposal!.choices.map((_choice, index) => `choice.${index + 1}`),
```

Comment:
Could we add a condition on top level to ignore everything if there is no proposal. else we have to check it everywhere

Reply:
the `!` is just for typescript, as this function is never called if proposal is missing, thanks to the check on line 19